cmd/traefik/traefik.go

```diff
@@ -90,7 +90,7 @@ Complete documentation is available at https://traefik.io`,
 }
 
 func runCmd(staticConfiguration *static.Configuration) error {
-	if err := setupLogger(staticConfiguration); err != nil {
+	if err := setupLogger(context.Background(), staticConfiguration); err != nil {
```

Comment:
Maybe I missed something, but is it intentional to not use the same context as the one declared few line below?